src/keys/bip39.jp.rs

```diff
@@ -3,9 +3,9 @@
 
 // https://raw.githubusercontent.com/bitcoin/bips/master/bip-0039/japanese.txt
 
-pub const JAPANESE: Wordlist = Wordlist {
-    separator: "　",
-    words: &[
+pub const JAPANESE: Wordlist = Wordlist::new_unchecked(
+    ' ', //'　', // U+3000 is normalized to U+20
```

Comment:
Why a comment here and not in english?

Reply:
Initially Japanese wordlist used U+3000 as separator, but we need to normalize it anyway when deriving a seed. U+3000 normalizes to U+0020 which is a regular ASCII space that is used by default in English wordlist.